daml/Marketplace/Custody/Model.daml

```diff
@@ -113,9 +115,17 @@ template LifecycleRequest
               election : Claim Observation Date Id -> Claim Observation Date Id -> Claim Observation Date Id
               election _ _ = deserialize choice -- will throw if the claims don't match
           Lifecycle.Result{remaining, pending} <- Lifecycle.lifecycle spot election (deserialize description.claims) t
-          (_, postAssetVersion) <- exercise versionsCid . AssetDescription.LookupOrInsert $ serialize remaining
-          payoutCids <- forA pending \(qty, asset) -> do
-            payoutDepositCid :: _ <- exercise safekeepingDepositCid $ AssetDeposit_Split [qty * assetDeposit.asset.quantity]
+          let serializedClaims = serialize remaining
```

Comment:
Small detail, but I'd rename this something like `remaining'` just to make it clear it's the remaining claims, not the original ones.

Reply:
Done.